src/javascripts/components/forms/createMenuItemForm.js

```diff
@@ -0,0 +1,37 @@
+import selectIngredients from '../menu/selectIngredient';
+
+const createMenuItemForm = () => {
+  document.querySelector('#stage').innerHTML = '';
+  document.querySelector('#view').innerHTML = '';
+
+  document.querySelector('#form-container').innerHTML = `
+  <form id="create-menu-item-form" class="mb-4">
```

Comment:
Since your id here starts with `create-menu-item` , if you click somewhere in the form it will submit because of [this code here](https://github.com/nss-evening-cohort-14/michelin-star-restaurant-dashboard-michelin-star-restaurant-dashboard-1/pull/63/files#diff-ab5e1cc150cfaa631fedac7c7dd62abf3193885ad54ff734b799cc96c121ce2bR87)

Here's an example of this bug

https://user-images.githubusercontent.com/31781724/111875049-bc5f5a80-8965-11eb-97e7-19d8d3be1152.mov